apps/frontend/src/libs/components/modal/modal.module.css

```diff
@@ -0,0 +1,66 @@
+.modal-container::backdrop {
+	background-color: var(--color-overlay);
+}
+
+.modal-container {
+	position: relative;
+	box-sizing: border-box;
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+	width: 800px;
+	height: 190px;
+	padding: 24px;
+	background: var(--color-background-secondary);
+	border: 1px solid var(--color-border-secondary);
+	border-radius: 8px;
+}
+
+.modal-header {
+	display: flex;
+	flex-direction: column;
+	gap: 24px;
+	width: 100%;
+}
+
+.modal-header-title {
+	font-family: Inter, sans-serif;
+	font-size: 20px;
+	font-weight: 600;
+	line-height: 26px;
+	color: var(--color-text-primary);
+}
+
+.modal-body {
+	width: 100%;
+	font-family: Inter, sans-serif;
+	font-size: 16px;
+	font-weight: 400;
+	line-height: 24px;
```

Comment:
better to set the line-height using either a percentage (150%) or a unitless number (1.5). Please check this in Figma. It was also mentioned in one of our last daily meetings